src/js/modules/supervisors/FileManager.ts

```diff
@@ -171,6 +166,23 @@ class FileManager {
     }
   }
 
+  /**
+   * receive an Error array, and return one error with all error messages.
+   * @param errors
+   */
+  compactErrors(errors: Error[][]): Error {
```

Comment:
Seems like this could just be `map` + `join`.

Reply:
I don't understand the comment, should I change the name? Should I change the logic? Should I use the map and join y instance to this function?

Reply:
Done.